components/bl-audio-recorder/src/helpers.js

```diff
@@ -5,7 +5,7 @@ export const download = (blob, fileName, fileType) => {
 
   link.style.display = 'none';
   link.href = URL.createObjectURL(blob);
-  link.download = `${ fileName }.${ fileType }`;
+  link.download = `${ fileName }.${ fileType.toLowerCase() }`;
```

Comment:
Why do we need it?

Reply:
To avoid extensions in upper case. I'm not sure but it could be a problem for some audio players.